File: 01.ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Exercises/src/P06_IntervalOfNumbers.java
import java.util.Scanner;
public class P06_IntervalOfNumbers {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        int start = Integer.parseInt(console.nextLine());
        int end = Integer.parseInt(console.nextLine());

        if(start < end) {
            for(int i = start; i <= end; i++) {
                System.out.println(i);
            }
        } else {
            for(int i = end; i <= start; i++) {
                System.out.println(i);
            }
        }
    }
}

File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Exercises/P06_IntervalOfNumbers/P06_IntervalOfNumbers.cs
using System;

namespace P06_IntervalOfNumbers
{
    class P06_IntervalOfNumbers
    {
        static void Main(string[] args)
        {
            int start = int.Parse(Console.ReadLine());
            int end = int.Parse(Console.ReadLine());

            if (start < end)
            {
                for (int i = start; i <= end; i++)
                {
                    Console.WriteLine(i);
                }
            }
            else
            {
                for (int i = end; i <= start; i++)
                {
                    Console.WriteLine(i);
                }
            }
        }
    }
}
